### DISCUSSION TITLE
Docker image issue

### DISCUSSION THREAD

**Discussion by joaobordignon** [firefly-iii/firefly-iii#5656] [Firefly III] [answered]
Hi there,

I'm not able to run the latest image from the docker repo,
the message below keep showing up even after update the image on my .yml file.

> The Firefly III image has moved
> For the latest version of Firefly III, use fireflyiii/core:latest
> 
> For the develop (unstable) version of Firefly III, use fireflyiii/core:develop
> 
> My apologies for the inconvenience.
> 
> All the best,
> James

I used the DB tech instruction to try to run it.

Thanks!

**Comment by JC5** [comment, marked as answer]
Stop and remove the container, then use `docker image prune` to remove all dangling images.

**Comment by joaobordignon** [comment]
Thanks I did it, and it worked after I replaced the .env and docker-compose.yml files in the server.

Thank you again!
